parser/proto.y

```diff
@@ -427,14 +427,14 @@ messageLiteralFieldEntry : messageLiteralField {
 	}
 
 messageLiteralField : messageLiteralFieldName ':' value {
-		if $1 != nil {
+		if $1 != nil && $2 != nil {
```

Comment:
This was the first buggy production. This corresponds to the panic in newly added test case "issue-196-a".